### ISSUE TITLE
Shared link page UI isn't optimal when opend in a mobile browser

### ISSUE THREAD

**Issue by simon-vajda** [immich-app/immich#8768]
### The bug

I really like that I can share an album with anyone using a link, however most of the time my friends open it on their phones where the UI isn't quite as good as in desktop mode or in the app. I feel like that with a couple of minor changes it could be a lot better. Here are my ideas:

Changes:
- I removed the scrollbar on the right. It's hard to use on mobile and it takes up a lot of space.
- Decreased the title font size from `3.75rem` to `2rem`. It can look wierd with the larger font size when the title is long.
- The logo should either 
  - appear without the title just like on the homepage
  - **OR** it can be displayed with the name to promote it to new users who haven't used immich yet but then it should have more width.

| Before  | After    |
| -------- | ------- |
| ![before](https://github.com/immich-app/immich/assets/43452607/95846e23-44ef-4288-b0d2-74f313bcd320)| ![after](https://github.com/immich-app/immich/assets/43452607/fd6236bd-5e3e-4ed9-92eb-da85ae29fadc)|

### The OS that Immich Server is running on

Unraid (docker)

### Version of Immich Server

v1.101.0

### Version of Immich Mobile App

-

### Platform with the issue

- [ ] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
-
```

### Your .env content

```Shell
-
```

### Reproduction steps

```bash
1. Create a shared link to an album, and open it in a mobile browser where you are not logged in.
```

**Comment by flightmansam** [comment]
This has been annoying me too!

**Comment by Snowknight26** [comment]
It this still an issue? I believe it was fixed with #9346.

**Comment by flightmansam** [comment]
Yep!
